SparrowImplicitLib/par/config.spr

```diff
@@ -1,27 +1,34 @@
 module par.config
 
+import std.compilerInfo
 import std.ptr
 
-datatype NativeThreadHandle = Byte Ptr  // Opaque type
+[ct] if platformName == "Linux"
+    datatype NativeThreadHandle = ULong
```

Comment:
I think you also need to add the `>>` operator for the new type (if I remember correctly it will not be autogenerated, so you can't print a NativeThreadHandle)